playground/package.json

```diff
@@ -12,6 +12,7 @@
     "@microsoft/tsdoc": "0.14.1",
     "@types/react-dom": "16.9.3",
     "@types/react": "16.9.11",
+    "react": "18.2.0",
```

Comment:
```suggestion
```

React should come from CloudFlare.

This is also a different major version than the `@types` package.